NDTensors/Project.toml

```diff
@@ -18,6 +18,7 @@ GPUArraysCore = "46192b85-c4d5-4398-a991-12ede77f4527"
 HalfIntegers = "f0d1745a-41c9-11e9-1dd9-e5d34d218721"
 InlineStrings = "842dd82b-1e85-43dc-bf29-5d0ee9dffc48"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
```

Comment:
Not needed now.

Reply:
Actually, I am still using the `@capture` macro inside the `@maybe_grow` macro implementation, just not using it for checking the validity of the expression.

Reply:
Oh right, never mind then.